libctru/source/services/fs.c

```diff
@@ -9,20 +9,30 @@
 #include <3ds/env.h>
 #include "../internal.h"
 
+#define FS_MAX_ARCHIVE_HANDLES 32
```

Comment:
When I ran tests, I was able to create 27 archives before getting out of resource errors, and I assume the remaining 5 are in use by other processes and the SDMC driver.